ui/test/nuxeo-tooltip.test.js

```diff
@@ -0,0 +1,50 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html, fixture, flush, isElementVisible } from '@nuxeo/testing-helpers';
+import '../widgets/nuxeo-tooltip.js';
+
+suite('nuxeo-tooltip', async () => {
+  test('Should not show a paper-tooltip if it has hidden atrribute set to true', async () => {
```

Comment:
typo on _attribute_. I would simplify this to:
```suggestion
  test('Should hide paper-tooltip when hidden attribute is set', async () => {
```

Reply:
Just my 2cents...
If I'm seeing the code in a correct way, it doesn't _hide_ the tooltip, it doesn't add it to the DOM.

Reply:
Agree, so maybe a more clear description would be:
```suggestion
  test('Should not add paper-tooltip to the dom when hidden attribute is set', async () => {
```